app/web/features/communities/CommunityPage/CommunityPage.tsx

```diff
@@ -64,7 +63,7 @@ export const useCommunityPageStyles = makeStyles((theme) => ({
 export default function CommunityPage() {
   const classes = useCommunityPageStyles();
 
-  return (
+  return null; /*(
```

Comment:
Why is this now returning nothing for the community page? 🤔

Reply:
I'm not up to it yet :P